sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveSessionCatalog.scala

```diff
@@ -212,3 +240,31 @@ private[sql] class HiveSessionCatalog(
     "histogram_numeric"
   )
 }
+
+private[sql] object HiveSessionCatalog {
+
+  def apply(
```

Comment:
Is this apply used anywhere other than in `clone`? if not, then there is no point creating an object with an `apply`.